uflash.py

```diff
@@ -72,8 +75,53 @@ def find_microbit():
 
     Returns None if no micro:bit is found.
     """
-    # TODO: FUNKYHAT'S WORK HERE!!!!!
-    pass
+    if os.name == 'posix':
+        mount_output = check_output('mount').splitlines()
+        # (almost) list the mounted volumes.
```

Comment:
This should probably be expanded. On OSX some of the output of `mount` looks like
/dev/sda on / (stuff)

and other bits looks like
map /mapper/something on /mapped/place (stuff again)

Which means field [2] isn't always the actual mount point, but we don't care because MICROBIT will never be a mapped drive, so just looking at [2] is good enough.